src/components/ProfileDetailsActionMenu/ProfileDetailsActionMenu.js

```diff
@@ -21,12 +22,16 @@ export const ProfileDetailsActionMenu = ({
     onToggle();
   };
 
+  const stripes = useStripes();
+  const hasViewPerms = stripes.hasPerm('settings.data-export.view') && !stripes.hasPerm('module.data-export.enabled');
```

Comment:
With this implementation user will be able to only see settings if they don't have access to main Data Export app. But what about users who need access to main app, but not edit settings?
I think it's better to create two permissions: `settings.data-export.view` and `settings.data-export.edit` and check if user doesn't have `settings.data-export.edit` then disable those actions

Reply:
We already have edit permission `Settings (Data export): Can view, create, edit, and remove` called `settings.data-export.enabled`. It's strange name I know, but it already implemented.